saleor/order/models.py

```diff
@@ -315,6 +311,16 @@ class Meta:
     def __str__(self):
         return self.product_name
 
+    def save(self, *args, **kwargs):
+        if not self.pk:
```

Comment:
Every additional logic in `model.save()` adds magic to the behavior and here we're also saving a different model under the hood. From what I see we already have the logic to update the weight in `add_variant_to_order`, so why do we need it here as well?

Reply:
Ok, I'll change that place. And `conftest` and `random_data`, where `order.lines.create` is used.